.github/workflows/publish-packages.yaml

```
@@ -27,17 +27,17 @@ jobs:
           ref: ${{ github.head_ref }}
 
           # this is important so git log has the whole history
-          fetch-depth: "0"
+          fetch-depth: '0'
 
       - name: Setup git
         run: |
           git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config --local user.name "github-actions[bot]"
 
-      - name: "Use Node.js 20"
+      - name: 'Use Node.js 20'
```

I'd say it's self-explanatory without the name

```suggestion
```

alternatively

```suggestion
      - name: 'Use Node.js'
```